Assets/Tests/Runtime/NetworkedMessageChannelTests.cs

```diff
@@ -0,0 +1,204 @@
+using System.Collections;
+using Unity.Multiplayer.Samples.BossRoom.Shared.Infrastructure;
+using Unity.Netcode;
+using UnityEngine;
+using Unity.Netcode.TestHelpers.Runtime;
+using UnityEngine.Assertions;
+using UnityEngine.TestTools;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Tests.Runtime
+{
+    public class NetworkedMessageChannelTests : NetcodeIntegrationTest
```

Comment:
You're testing for successful cases, you'd normally want to also test for edge cases and failures.

Reply:
good point, will add more coverage